src/alexjs/alex.service.ts

```diff
@@ -22,8 +22,14 @@ export class AlexService {
     const alexMatch = alex.markdown(messageText, alexWhitelist as alex.Config)
       .messages;
 
+    let blockedLinks = 0;
+
+    const linkRegex = /(([a-z]+:\/\/)?(([a-z0-9-]+\.)+([a-z]{2,3}|aero|arpa|coop|info|jobs|museum|name|nato|travel|local|internal))(:[0-9]{1,5})?(\/[a-z0-9_\-.~]+)*(\/([a-z0-9_\-.]*)(\?[a-z0-9+_\-.%=&amp;]*)?)?(#[a-zA-Z0-9!$&'()*+.=-_~:@/?]*)?)(\s+|$)/gi;
```

Comment:
we should discuss if ne weed to modifiy the Regex, or if the "Becca-One" works for Eddiebot as well.

Reply:
I am not sure, tbh links are difficult to detect correctly, and I don't want a way for people to get around the bot. I think maybe we detect giphy and tenor links only in the regex for now and we can add more to the regex as needed?

Something like this `[a-z.]*(tenor.com|giphy.com)/?[a-z]*`

<img width="1680" alt="Screenshot 2021-08-21 at 14 11 39" src="https://user-images.githubusercontent.com/624760/130322917-1c8704ff-8156-4e93-8f58-9ae34ab95f49.png">

Reply:
We shouldn't check for GIFs *only*, we should also check for all kinds of links.

Reply:
I'm wondering... could we use this instead of a regex? - 
https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=embeds